### PR TITLE
Launch MIX servers through a yarn cluster manager

### PR DESCRIPTION
**Pull request by maropu**
Implemented from #236.

To launch MIX servers via YARN, you set "1" at `MIXSERV_YARN_ENABLED` in `conf/mixserv_env.sh` and just say `./bin/mixserv_cluster.sh start`. '-mix yarn://[YARN APPLICATION MASTER IP' in training UDTF functions gives you model-mixing in the MIX servers.

### CODE REVIEW

File: bin/mixserv_daemon.sh [hunk 1]
```diff
@@ -119,13 +153,13 @@ case $1 in
     if [ -f $MIXSERV_PID_FILE ]; then
```
Comment (myui) [inline bin/mixserv_daemon.sh]: better to use `yarn application -kill` for stopping yarn application.

https://hadoop.apache.org/docs/r2.4.1/hadoop-yarn/hadoop-yarn-site/YarnCommands.html#application

File: bin/mixserv_daemon.sh [hunk 2]
```diff
@@ -134,14 +168,14 @@ case $1 in
...
```
Comment (myui) [inline bin/mixserv_daemon.sh]: better to apply `yarn application -status` for YARN mode.

https://hadoop.apache.org/docs/r2.4.1/hadoop-yarn/hadoop-yarn-site/YarnCommands.html#application

Comment (maropu) [inline bin/mixserv_daemon.sh]: I think that the current implementation does the same thing with `yarn application -status` when SIGTERM/SIGINT received.
https://github.com/myui/hivemall/pull/246/files#diff-b1e95d2afdb57b4e66a66403f081634fR339

File: bin/mixserv_daemon.sh (hunk 1 above)
Comment (maropu) [inline bin/mixserv_daemon.sh]: I think that the current implementation does the same thing with `yarn application -status` when SIGTERM/SIGINT received.
https://github.com/myui/hivemall/pull/246/files#diff-b1e95d2afdb57b4e66a66403f081634fR339

**Pull request closed without merging**